validate/validate.go

```diff
@@ -129,6 +132,25 @@ func PolicyToOptions(policy *cpb.Policy) (*Options, error) {
 	return opts, nil
 }
 
+func byteCheckRtmr(option, field string, size int, given, required [][]byte) error {
+	if len(required) == 0 {
+		return nil
+	}
+	if len(required) != 4 {
```

Comment:
4 should be in a defined constant like rtmrCount

Reply:
Done.